src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillHttpTransport.php

```diff
@@ -49,7 +49,8 @@ protected function doSendHttp(SentMessage $message): ResponseInterface
                 'to' => array_map(function (Address $recipient): string {
                     return $recipient->getAddress();
                 }, $envelope->getRecipients()),
-                'from_email' => $envelope->getSender()->toString(),
+                'from_email' => $envelope->getSender()->getAddress(),
+                'from_name' => $envelope->getSender()->getName(),
```

Comment:
A sender in the envelope does not have a name. If that works, it is by chance and won't work anymore in 5.0 anyway. I haven't time right now to investigate, but we need to find another way.

Reply:
Do you mean that https://github.com/symfony/symfony/blob/017420b5119578ae45adb58dc9118c701b4d2b7f/src/Symfony/Component/Mailer/Envelope.php doesn't set the name when creating new Address objects?

Also i just tested this and in gmail i see correct name i set via Address object like:
`->from(new Address('test@example.nl', 'Testing'))`

I will get `Testing` as name in my mail

Reply:
Is there anything i can do?

Reply:
I have looked again and i'm doing the same thing as in Sendgrid see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Mailer/Bridge/Sendgrid/Transport/SendgridApiTransport.php#L78 the only difference here is we check if name is not empty i can also add that to my pull request?

I also see the same code is used in Mailchimp transporter which makes sence as Mandrill and Mailchimp are one see code https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillHttpTransport.php#L54 do you want me to make changes here as well?

Reply:
It now works as expected (refs #37580).